src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -50,3 +50,70 @@ def test_get_self_cal_last_date_and_time():
 
 def test_self_cal(session):
         session.self_cal()
+
+
+def test_create_waveform(session):
+    waveform_data = [0.000000, 0.049068, 0.098017, 0.146730, 0.195090, 0.242980, 0.290285, 0.336890, 0.382683, 0.427555]
+    session.output_mode = nifgen.OutputMode.NIFGEN_VAL_OUTPUT_ARB  # TODO(Jaleel): name to change per #553
+    session.configure_arb_waveform(session.create_waveform_f64(waveform_data), 1.0, 0.0)
```

Comment:
this test is supposed to test create waveform. Don't call configure_arb_waveform as well - do that in a separate test or rename this test.